### DISCUSSION TITLE
Ajuda kodi, ott por favor

### DISCUSSION THREAD

**Discussion by fcprush** [LITUATUI/M3UPT#55] [General]
Olá boa noite! 
Por favor alguém me pode ajudar a ver TV no kodi?? 
Eu instalei o Ott mas o código de backup não funciona! 
Alguém me ajuda por favor???

**Comment by LITUATUI** [comment]
Tenta de novo, o código funciona.

**Reply by fcprush** [reply]
Não da! Já experimentei várias vezes 😥

**Reply by fcprush** [reply]
Eu meto o código, e não faz nada, nem me pede a password nem nada 😓

**Reply by LITUATUI** [reply]
Acabei de experimentei o código no telemóvel e funciona. Fiz copy-paste. Tem atenção às minúsculas/maiúsculas.

**Reply by fcprush** [reply]
Já consegui! Desinstalei o Ott qe tinha e instalei outra versão e já deu 😁 obrigado 😎👍🏼
